src/Core_Command.php

```diff
@@ -1150,21 +1176,26 @@ function update_db( $_, $assoc_args ) {
 		if ( $network ) {
 			$iterator_args = array(
 				'table' => $wpdb->blogs,
-				'where' => array( 'spam' => 0, 'deleted' => 0, 'archived' => 0 ),
+				'where' => array(
+					'spam'     => 0,
+					'deleted'  => 0,
+					'archived' => 0,
+				),
 			);
-			$it = new \WP_CLI\Iterators\Table( $iterator_args );
-			$success = $total = 0;
-			$site_ids = array();
-			foreach( $it as $blog ) {
+			$it            = new \WP_CLI\Iterators\Table( $iterator_args );
+			$success       = 0;
+			$total         = 0;
+			$site_ids      = array();
+			foreach ( $it as $blog ) {
 				$total++;
 				$site_ids[] = $blog->site_id;
-				$url = $blog->domain . $blog->path;
-				$cmd = "--url={$url} core update-db";
+				$url        = $blog->domain . $blog->path;
+				$cmd        = "--url={$url} core update-db";
 				if ( $dry_run ) {
 					$cmd .= ' --dry-run';
 				}
 				$process = WP_CLI::runcommand( $cmd, array( 'return' => 'all' ) );
-				if ( 0 == $process->return_code ) {
+				if ( 0 === (int) $process->return_code ) {
```

Comment:
Probable bug. This is a dangerous comparison as any non-numeric string will cast to `0`.
```suggestion
				if ( '0' === $process->return_code ) {
```